app/utils/filesystem/storage.js

```diff
@@ -7,11 +7,13 @@ import * as fs from 'fs';
 import * as os from 'os';
 import * as path from 'path';
 import recursive from 'recursive-readdir';
-import { shell } from 'electron';
+import { shell, remote } from 'electron';
+import Papa from 'papaparse';
```

Comment:
Papaparse looks like a great library to use. Don't know why I didn't go with it last year